main.go

```diff
@@ -17,6 +17,8 @@ func main() {
 	flag.IntVar(&conf.StopAt, "stop-at", conf.StopAt, "PR number to stop at")
 	flag.BoolVar(&conf.IncludeCommits, "include-commits", conf.IncludeCommits, "Include commit messages")
 	flag.BoolVar(&conf.SinceLatestRelease, "since-latest-release", conf.SinceLatestRelease, "Stop at latest release's commit")
+	flag.BoolVar(&conf.IncludeAuthor, "include-author", conf.IncludeAuthor, "Include author of PR in message")
+	flag.StringVar(&conf.GitHubToken, "github-token", os.Getenv("GITHUB_TOKEN"), "Github Token.  (Defaults to env GITHUB_TOKEN)")
```

Comment:
I like the flag, but I'd prefer not to expose the token via the `--help` text (because it will print the default).

Reply:
Makes sense, changed to hide that during `--help` and set during actual parsing